src/components/Modal/Modal.module.scss

```
@@ -69,12 +77,8 @@
         max-width: min(var(--rui-local-max-width), #{map.get(theme.$sizes, auto, max-width)});
     }
 
-    .isRootPositionCenter {
-        top: 50%;
-        transform: translate(-50%, -50%);
-    }
-
     .isRootPositionTop {
+        position: sticky;
         top: var(--rui-local-outer-spacing);
```

I think this should do the job:

```suggestion
        top: var(--rui-local-outer-spacing);
        bottom: auto;
```